### PR TITLE
do not fail when error message is too long

### PR DESCRIPTION
**Pull request by grosser**
@sandlerr ... also fixing up test coverage

https://zendesk.airbrake.io/projects/95346/groups/1714212048688589520

### CODE REVIEW

File: plugins/jenkins/app/models/samson/jenkins.rb [hunk 1]
```diff
@@ -6,9 +6,25 @@ class Jenkins
     URL = ENV['JENKINS_URL']
     USERNAME = ENV['JENKINS_USERNAME']
     API_KEY = ENV['JENKINS_API_KEY']
+    ERROR_COLUMN_LIMIT = 255
 
     attr_reader :job_name, :deploy
 
+    def self.deployed!(deploy)
+      return unless deploy.succeeded?
+      deploy.stage.jenkins_job_names.to_s.split(/, ?/).map do |job_name|
```
Comment (sandlerr) [inline plugins/jenkins/app/models/samson/jenkins.rb]: the old code made sure `jenkins_job_names` was present, should that still happen?

**Comment by sandlerr** [comment]
:+1: on green

File: plugins/jenkins/app/models/samson/jenkins.rb (hunk 1 above)
Comment (grosser) [inline plugins/jenkins/app/models/samson/jenkins.rb]: empty string will be split into nothing -> .each will do nothing

**Comment by sandlerr** [comment]
codeclimate and travis were both sad :/

**Pull request merged**